product_packaging_unit_price_calculator/tests/test_product_price_packaging_qty.py

```diff
@@ -0,0 +1,50 @@
+from odoo.tests.common import Form, SavepointCase
+
+
+class TestProductPricePackagingQty(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product = cls.env.ref("product.product_product_1")
+        cls.wizard = cls.env["product.package.price.wizard"]
+        cls.pkg_box = cls.env["product.packaging"].create(
+            {"name": "Box", "product_id": cls.product.id}
+        )
+        # Qty is not added on create because it breaks on Travis with
+        # packaging_uom installed
+        cls.pkg_box.qty = 50
+        cls.pkg_big_box = cls.env["product.packaging"].create(
+            {"name": "Big Box", "product_id": cls.product.id}
+        )
+        cls.pkg_big_box.qty = 200
+        cls.pkg_pallet = cls.env["product.packaging"].create(
+            {"name": "Pallet", "product_id": cls.product.id}
+        )
+        cls.pkg_pallet.qty = 2000
+        cls.wizard_1 = cls.wizard.create({"product_id": cls.product.product_tmpl_id.id})
+        cls.supplier = cls.env["res.partner"].create({"name": "Good Supplier"})
```

Comment:
do you really need to create a new partner? As you don't have special config for it I'd use an existing one from demo data.

Reply:
Nope, changed